src/queries/find/using.ts

```diff
@@ -4,11 +4,12 @@ import { Limit } from './limit';
 import { Order } from "./order";
 import { DocumentClient } from "aws-sdk/clients/dynamodb";
 import { Resume } from "./resume";
+import { StronglyConsistent } from "./stronglyConsistent";
 
 export class Using
 {
-    constructor(private manager:IDynamoDbManager, 
-                private type:{new(...args: any[])}, 
+    constructor(private manager:IDynamoDbManager,
+                private type:{new(...args: any[])},
```

Comment:
I think we need `stronglyConsistent?:boolean` in the param.

Reply:
Good catch!